modules/context/context.go

```diff
@@ -67,8 +67,9 @@ type Context struct {
 	IsSigned    bool
 	IsBasicAuth bool
 
-	Repo *Repository
-	Org  *Organization
+	ContextUser *user_model.User
```

Comment:
What do you think about renaming `Context.User` to `Context.Doer`?
Later, once devs are accustomed to the change, `Context.ContextUser` can become `Context.User`, which seems both cleaner and more explicit.

Reply:
I also agree that `ContextUser` is a very confusing name. It would be misused a lot in future. `Doer` seems not ideal, because in many places, doer is current context user, for example: the user edits a comment is a doer.

I think we can have some new names for it:

* PathUser
* VisitedUser
* AccssedUser
* OrgUser

Which one do you like, or could there be more better names?

Reply:
And we need more comments ........... 😊

Reply:
I added a PR for that: #19161

Reply:
I'm for Doer - it's already used but not consistendly (as func arg name ...)